app/assets/stylesheets/base/boxes.scss

```diff
@@ -0,0 +1,40 @@
+@import '../vars/_variables.scss';
+
+@mixin box($this: $this, $color-1: $smoke-90, $color-2: $smoke-100) {
+  border-color: $color-2;
+
+  &#{$this}--filled {
+    background-color: $color-1;
+  }
+
+  @for $i from 1 through 3 {
+    &#{$this + '--level-' + $i} {
+      box-shadow: #{pow(2, $i) + px} #{pow(2, $i) + px} 0 $color-2;
```

Comment:
What does `pow` stand for?

Reply:
It's a helper (see [_utils.scss](https://github.com/thepracticaldev/dev.to/pull/6334/files#diff-7de1ce7b96da3715f4ff4134f82180f8R13-R25) - I've described it there) to help with power function in math calculations in SCSS (unfortunately there's no other easy way to do it in SCSS yet). Its basically equivalent to `2^$i` (two to the power of `$i`).